build_docs.sh

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
+# To build manually run "sudo sh build_docs.sh"
```

Comment:
As this is a seldom used file I think it would be good to add some high level explanation about the SourceKitten and Jazzy interaction, e.g.

```
As MSAL is a mixed Objective-C and Swift project we need to use SourceKitten to
first extract the doc comments from the source before building with Jazzy. For more
details see https://github.com/realm/jazzy?tab=readme-ov-file#mixed-objective-c--swift
```

Reply:
added